src/routes/[deviceUID]/dashboard/+page.server.ts

```diff
@@ -39,6 +40,9 @@ export async function load({ params }) {
     .then((responses) => {
       const [envVarResponse, eventsResponse] = responses;
       const serialNumber = envVarResponse.environment_variables._sn;
+      if (envVarResponse.environment_variables._air_indoors === 1) {
```

Comment:
you were very close, the code check actually needs to be 
```suggestion
      if (envVarResponse.environment_variables._air_indoors === '1') {
```